v3/src/components/map/map-types.ts

```diff
@@ -46,7 +47,15 @@ export const
 
 
   kLatNames = ['latitude', 'lat', 'latitud', 'breitengrad', '緯度', 'عرض جغرافیایی'],
-  kLongNames = ['longitude', 'long', 'lng', 'lon', 'longitud', 'längengrad', '経度', 'طول جغرافیایی']
+  kLongNames = ['longitude', 'long', 'lng', 'lon', 'longitud', 'längengrad', '経度', 'طول جغرافیایی'],
+
+  kPinLatNames = ['pinlatitude', 'pinlat', 'pinlatitud', 'pinbreitengrad', 'pin緯度', 'pinعرض جغرافیایی'],
+  kPinLongNames = [
+    'pinlongitude', 'pinlong', 'pinlng', 'pinlon', 'pinlongitud', 'pinlängengrad', 'pin経度', 'pinطول جغرافیایی'
+  ],
```

Comment:
That's probably not the right solution for languages like Chinese that use a different alphabet, but it's probably the best we can do right now. Ultimately, this should be integrated into the rest of our localization system, but that's a project for another day. Maybe for now add a TODO comment about figuring out a better solution.